gamestonk_terminal.py

```diff
@@ -150,6 +149,37 @@ def view(l_args, s_ticker, s_start, s_interval, df_stock):
     plot_view_stock(df_stock.iloc[:, ln_col_idx], ns_parser.s_ticker)
 
 
+# ----------------------------------------------------- EXPORT -----------------------------------------------------
+def export(l_args, df_stock):
+    parser = argparse.ArgumentParser(prog='export', description='Exports the current dataframe to a file or stdout')
+    parser.add_argument('-f', "--filename", type=str, dest="s_filename", default=stdout, help="Where to save the export (stdout if unspecified)")
```

Comment:
As default, wouldn't be better to save for a file with the following default name:
datestart_dateend_stockname.csv ?

I can't see how it can be useful printing 100 days worth of data, if it makes sense.

Reply:
So the idea here was to enable a non-interactive, batch mode use of the terminal in line with other REPL-driven software like Python itself or the Unix shell. For instance, you could have a script like the following:
`#!/usr/bin/env -S python gamestonk_terminal.py
load -t GME -s $(date -d "7 days ago" '+%y%m%d')
export`
which could then be used in a shell pipeline. You might, say, gzip the output without having to write it to a temporary file first. So there's definitely a use case for having output to stdout I would say.

However, I agree that it may not make as much sense for the default behavior - I was just going by the typical Unix convention of having output go to stdout if an output filename isn't specified on the command line. It might be better to have a -s for stdout option or something like that.

The thing about using datestart and dateend as default filenames is that it's totally possible to load tickers without specifying dateend or datestart; dateend by default can just be now but I'm not sure what I would do for datestart. Or is that already calculated based on the earliest data returned from the API request?

Reply:
Yea, that makes sense.

We can go for the conventional way, and if in the future we think there's a better default way, we change it :)